en/advanced_config/prearm_arm_disarm.md

```diff
@@ -4,13 +4,47 @@ Vehicles may have moving parts, some of which are potentially dangerous when pow
 
 To reduce the chance of accidents, PX4 has explicit state(s) for powering the vehicle components:
 - **Disarmed:** There is no power to motors or actuators.
```

Comment:
Not entirely sure about this, because we're talking about states, not what you see in QGC. I've added a note though "Ground stations may display *disarmed* for both disarmed and pre-armed states.
  While not technically correct for pre-armed vehicles, it is "safe"."